io.github.bgrabitmap.LazPaint.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.bgrabitmap.LazPaint
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.freepascal
+command: lazpaint
+build-options:
+    no-debuginfo: true
```

Comment:
why removing the debug info?

Reply:
I will remove that as that was supposed to be removed. It was just used for getting stack trace for some program exception.